ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3551,24 +3551,123 @@ signal_list_t* create_operation_node(ast_node_t* op, signal_list_t** input_lists
     allocate_more_output_pins(operation_node, output_port_width);
     add_output_port_information(operation_node, output_port_width);
 
+    if ( (op->children[1]->type == NUMBERS) && (operation_node->type == SL || operation_node->type == SR || operation_node->type == ASR) ) {
```

Comment:
this will break off for variable shifting because of 

https://github.com/CAS-Atlantic/vtr-verilog-to-routing/pull/26/files#diff-ab25514ea7cdfbd91f356b27ad8c7ceeR3511-R3516

can you move the logic upwards a bit

Reply:
Sure, but we can not do this since we used input_pins of the operation_node, which are just defined a few lines before this logic, for output_pins definitions. Also, the definition of the variable shifting is right after this if. Did I get the point correctly?